.github/workflows/docs.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Update Docs'
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  docs:
+    name: Update Docs
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+      - name: Install Packages
+        run: npm install
+      - name: Build Docs
+        run: npm run docs
+      - name: Deploy 🚢
+        uses: peaceiris/actions-gh-pages@v3
```

Comment:
I'd really rather publish to a `ldapjs.github.io` repository instead of a `gh-pages` branch.

I have this action that does this for Hugo based projects -- https://github.com/jsumners/gh-action-hugo-deploy

It'd be easy to rewrite for this setup since we probably wouldn't even need Docker.

Reply:
Does that action have to use hugo? We could change it to that in another PR. We should move the CNAME to that repo then since most links still go to `ldapjs.org` instead of `ldapjs.github.io`.

We still need to figure out #679.

Reply:
I'm suggesting we create a new action in the same vein. That one is very much specific to Hugo.

For now, I've gone ahead and removed the custom domain from this repository.

I know. I just updated #679 with at least a little bit of movement.

Reply:
Looks like there are quite few actions that do that. The best one I am finding is [s0/git-publish-subdir-action](https://github.com/marketplace/actions/push-git-subdirectory-as-branch)

Reply:
Looks good to me. Seems it can be run totally within the action so we do not need to commit build artifacts to this repository.

Reply:
I used cpina/github-action-push-to-another-repository instead but we will need to add a secret with repo persmission to `API_TOKEN_GITHUB`

Reply:
Done.